### ISSUE TITLE
Can't go fullscreen on 0.83.12 version (Windows 9x)

### ISSUE THREAD

**Issue by Farooq87** [joncampbell123/dosbox-x#2427]
I just toggle fullscreen in menu bar and it gives no effect. Instead of fullscreen menu disappears and thats all. Same window resolution.

**Comment by Wengier** [comment]
I think this is likely mostly limited to Windows 9x 3dfx games, which do not support full-screen mode unless it is emulated via the Glide wrapper. Non-3dfx games will probably work fine in full-screen mode.

**Comment by Farooq87** [comment]
@Wengier. I run this my preinstalled Windows9x with 3dfx games in SVN Daum 2015 and it worked just fine in fullscreen, never folding to window. There might be some strange behaviour of Dosbox-X with particular wrappers in dosbox folder.

**Comment by Farooq87** [comment]
By the way I also cant switch on fullscreen if fullscreen resolution = original

**Comment by Farooq87** [comment]
I think problem might be connected with combination of fullscreen toggle. Its F11 + F, not Alt + Enter which is now internal guest OS command. Is it possible to fix it?

**Comment by Wengier** [comment]
Yes, it appears that the Daum build does allow fullscreen mode in low-level 3dfx emulation. I think there can eventually be an option to allow this in DOSBox-X too.

**Comment by Wengier** [comment]
I also want to add that the fullscreen restriction only occurs when the 3dfx emulation is set to the OpenGL mode. If you set voodoo_card=software, then you should be able to keep the 3dfx emulation in fullscreen mode anyway.

**Comment by Farooq87** [comment]
@Wengier
With voodoo_card=software emulation becomes extremely low although it allows to run 3dfx games in fullscreen
